src/makeCalculator.js

```diff
@@ -3,8 +3,42 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
+function makeCalculator(operation, value) {
   // write code here
+
+  const calculator = {
+    operate(callback, num) {
+      callback.call(this, num);
+
+      return this;
+    },
+
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+
+    result: 0,
+
+    add: function (number) {
```

Comment:
[CODE KNOWLEDGE] - For the `add` method, you can use the method shorthand. Replace `add: function (number) {` with `add(number) {`.